packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-drilldown.vue

```diff
@@ -148,6 +149,7 @@
 						:model-configurations="filteredModelConfigurations"
 						:mmt="configuredMmt"
 						:mmt-params="mmtParams"
+						:error-empty="true"
```

Comment:
Probably double check with @shawnyama here, it seems to me like we are unnecessarily bubbling up `errorEmpty`, which is to say start lower and thus not needed for
- tera-parameter-table
- tera-initial-table
- tera-model-config-drilldown

Arguably we don't need the props for
- tera-initial-entry
- tera-parameter-entry 
Either, but I am a bit out of the look what exactly what is the change request at hand ...